### PR TITLE
_G leaks fix and french translation update

### PR DESCRIPTION
**Pull request by mrbuds**
https://github.com/Vysci/LFG-Bulletin-Board/commit/90e3435856fa33f543ff3d576e31866659542062 fixes potential taints and breaking other addons
for https://github.com/Vysci/LFG-Bulletin-Board/commit/15085f0264696d9a2b56132bbddaab12f510b565 i used names found on wowhead

### CODE REVIEW

File: LFGBulletinBoard/Dungeons.lua [hunk 1]
```diff
@@ -168,31 +168,31 @@ function GBB.GetDungeonNames()
...
+			["EYE"] = 	"L'Œil du cyclone",
 			["ZA"] = 	"Zul-Aman",
-			["HYJAL"] = "The Battle For Mount Hyjal",
-			["BT"] = 	"Black Temple",
-			["SWP"] = 	"Sunwell Plateau",
+			["HYJAL"] = "Sommet d'Hyjal",
+			["BT"] = 	"Temple noir",
+			["SWP"] = 	"Plateau du Puits de soleil",
 			["ONY"] = 	"Repaire d'Onyxia (40)",
 			["MC"] = 	"Cœur du Magma (40)",
```
Comment (Vysci) [inline LFGBulletinBoard/Dungeons.lua:197]: You are overwriting English names. You need to put all the french names under a new variable

**Review by Vysci — CHANGES_REQUESTED** [review]

File: LFGBulletinBoard/Dungeons.lua (hunk 1 above)
Comment (mrbuds) [reply LFGBulletinBoard/Dungeons.lua:197]: Sorry but i don't get it, this fix english names in the dungeonNamesLocales.frFR table, dungeonNamesLocales.deDE / esMX / ruRU doesn't use english names

Comment (Vysci) [reply LFGBulletinBoard/Dungeons.lua:197]: oh you are right, my bad I didn't realize it was already in the french table

**Comment by Vysci** [comment]
Can you pull the latest changes, which include heroic filtering and add French localization to the newly introduced variables?

**Comment by mrbuds** [comment]
I have rebase the PR, fixed a new global you added and removed Tool.CopyPast, i saw it wasn't using correct backdrop template, then started fixing a few bugs it has, before i saw it wasn't use anywhere.

About the new localization variables, i don't play the game in french myself but i play on a french server, i wanted to make the addon work better on french servers but i don't have time & will to translate the whole UI, your last heroic changes should work correctly without an update in french localization.

**Pull request merged**